backend/cybnetics/resources/users.py

```diff
@@ -1,10 +1,46 @@
+import jwt
+import datetime
+from flask import current_app
+from flask_bcrypt import Bcrypt
 from .db import db
 
+
+bcrypt = Bcrypt(current_app)
+
+
 def create(username, password):
-    pass
+
+    users = db.users
+    is_user = users.find_one({'username': username})
+
+    if is_user is None:
+        hash_password = bcrypt.generate_password_hash(password)
+        users.insert({'username': username, 'password': hash_password.decode('utf-8')})
+        exp = datetime.datetime.utcnow() + datetime.timedelta(hours=24)
+        token = jwt.encode({'username': username, 'exp': exp}, current_app.config['SECRET'])
+        message = {'token': token.decode('utf-8')}
+        return message, 201
+    else:
+        message = {'status': '401','message': 'User already exists.'}
```

Comment:
(a bit of a nit pick, i can just handle this later if you want)
much better, the stuff is a bit more separated, but now we have some http stuff leaking in here (http error codes and the json formatting of the response). the idea behind this separation is that these resources would be more like a python library, so we can just have these functions either return just the jwt string or return an error/throw an exception.

Reply:
Not a nit pick; this makes a lot of sense; I was just unclear at first. I just implemented this in the recent commit "HTTP Isolation" (not sure if there is a way to tag commits).